app/frontend/src/features/search/LocationAutocomplete/LocationAutocomplete.test.tsx

```diff
@@ -71,4 +72,65 @@ describe("LocationAutocomplete component", () => {
       );
     });
   });
+
+  it("shows an error when submitting without selection an option", async () => {
+    const onChange = jest.fn();
+    renderForm(undefined, onChange);
+
+    const input = (await screen.findByLabelText(LOCATION)) as HTMLInputElement;
+    expect(input).toBeVisible();
+    userEvent.type(input, "test");
+    const submitButton = await screen.findByRole("button");
+    userEvent.click(submitButton);
+    expect(await screen.findByText(SELECT_LOCATION)).toBeVisible();
+  });
+
+  it("shows a default value and submits correctly when cleared", async () => {
+    const onChange = jest.fn();
+    renderForm(
+      {
+        name: "test location",
+        simplifiedName: "test location",
+        location: new LngLat(1, 2),
+      },
+      onChange
+    );
+
+    const input = (await screen.findByLabelText(LOCATION)) as HTMLInputElement;
+    expect(input).toBeVisible();
+    expect(input.value).toBe("test location");
```

Comment:
```suggestion
    expect(input).toHaveValue("test location");
```
In this case, the readability is negligible, but I think the custom matcher from React Testing Library is supposed to give clearer error messages when things fail so more that it's slightly better for debugging tests.

Reply:
Got a straggler to update, considering you've done it in the other tests :p